algoliasearch/pom.xml

```diff
@@ -25,6 +25,11 @@
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpclient</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.asynchttpclient</groupId>
+      <artifactId>async-http-client</artifactId>
+      <version>2.7.0</version>
```

Comment:
it's a good practice to put the version in a property, at the beginning of the file